packages/web3-eth2-beaconchain/src/types.ts

```diff
@@ -0,0 +1,36 @@
+import { altair, phase0 } from '@chainsafe/lodestar-types';
+import {
+    PrefixedHexString,
+    ValidTypesEnum,
+} from 'web3-utils/lib/types';
+
+export interface Web3EthOptions {
+    web3Client: string;
+    returnType?: ValidTypesEnum;
+}
+
+export type StateId =
```

Comment:
will be more good if this is string enum

Reply:
Fyi @luu-alex, you'll want an enum such as:

```javascript
export enum StateIds {
    Head = 'head',
    Genesis = 'genesis',
    ...
}
```